### PR TITLE
Initialize Wazuh plugin with custom logos

### LINKED ISSUE #4538: The custom logos are overlapped with Wazuh's at app start
**Description**
When the app is configured to use custom logos, the Wazuh logo appears for a short period of time at app start.

After that, the custom logo is used as it should.

**Preconditions**
1. Customs logos must be used, and properly configured.

**Steps to reproduce**
1. Open the app
2. Check that Wazuh's logo is shown for a brief period of time, before the custom logo is displayed.

**Expected Result**
1. The custom logo must be rendered every time. The Wazuh logo must not be seen for any period of time.

**Actual Result**
1. The Wazuh logo is shown for a short period of time, being overwritten by the custom logo later.

**Screenshots**

### PR DESCRIPTION
**Pull request by asteriscos**
# Description

This PR fixes an issue when custom logos are configured, Wazuh default logos load first when the plugin initializes and afterwards the custom logos are loaded. 

To fix this we created an endpoint to get the sidebar and the main custom logos configuration without the need to be authenticated. This endpoint is used just before the plugin initializes, so the proper logos are set up in the app mounting phase.

Closes #4538 

## Sample

Peek 2022-09-16 14-54

### CODE REVIEW

**Review by matiasmoreno876 — APPROVED** [review]
CR : LGTM ✅

File: server/routes/wazuh-api.ts [hunk 1]
```diff
@@ -153,4 +153,13 @@ export function WazuhApiRoutes(router: IRouter) {
   },
     async (context, request, response) => ctrl.isWazuhDisabled(context, request, response)
   );
+
+  // Return app logos configuration
+  router.get({
+    path: '/api/get-logos',
```
Comment (AlexRuiz7) [inline server/routes/wazuh-api.ts:159]: No need to add the action (HTTP verb) to the URL, it's already defined in the route (get).

It's not a good practice to do this, check guideline nr.1 from this [Best Practices for naming REST API endpoints](https://betterprogramming.pub/10-best-practices-for-naming-rest-api-endpoints-442ae592a3a0) guide.

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]
Code Review: 🔴 

Check suggestions

File: server/routes/wazuh-api.ts (hunk 1 above)
Comment (AlexRuiz7) [reply server/routes/wazuh-api.ts:159]: As @asteriscos is on holidays, I'll apply this change.

**Review by AlexRuiz7 — APPROVED** [review]
LGTM!

File: server/routes/wazuh-api.ts (hunk 1 above)
Comment (AlexRuiz7) [reply server/routes/wazuh-api.ts]: Solved in https://github.com/wazuh/wazuh-kibana-app/pull/4539/commits/0ace60f0912eb2fb9363999ba9180ffc135486f0

**Comment by AlexRuiz7** [comment]
Backports successfully merged

**Pull request merged**